jest/babel-test.js

```diff
@@ -0,0 +1,15 @@
+const babel = require("@babel/core");
+const importMetaBabelPlugin = require("./importMetaBabelPlugin");
+
+const code = `
+const modules = import.meta.glob("./components/**/*.tsx");
+`;
+
+const result = babel.transform(code, {
+  filename: "./src/file.ts",
+  plugins: [importMetaBabelPlugin],
+});
+
+console.log("\n--------------------------\n");
+
+console.log(result.code);
```

Comment:
We will need to delete this file eventually, but it's handy while we develop the plugin.

Reply:
Or maybe we should convert it into a jest test and actually assert to transformed code 🤔

Reply:
I removed the file, we could bring it back and add a test or wait until we create a package with this babel transform